[2024-10-24T19:47:24.039+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-24T19:47:24.584+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Assignment8_ETL_DAG.copyToSessionTimeStamp manual__2024-10-24T19:46:52.815149+00:00 [queued]>
[2024-10-24T19:47:24.706+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Assignment8_ETL_DAG.copyToSessionTimeStamp manual__2024-10-24T19:46:52.815149+00:00 [queued]>
[2024-10-24T19:47:24.710+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-24T19:47:25.429+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): copyToSessionTimeStamp> on 2024-10-24 19:46:52.815149+00:00
[2024-10-24T19:47:25.501+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Assignment8_ETL_DAG', 'copyToSessionTimeStamp', 'manual__2024-10-24T19:46:52.815149+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/Assignment8_ETL_DAG.py', '--cfg-path', '/tmp/tmpivjuj19a']
[2024-10-24T19:47:25.526+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask copyToSessionTimeStamp
[2024-10-24T19:47:25.542+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2333) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-24T19:47:25.549+0000] {standard_task_runner.py:63} INFO - Started process 2392 to run task
[2024-10-24T19:47:26.380+0000] {task_command.py:426} INFO - Running <TaskInstance: Assignment8_ETL_DAG.copyToSessionTimeStamp manual__2024-10-24T19:46:52.815149+00:00 [running]> on host d99256b0cb87
[2024-10-24T19:47:27.300+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Assignment8_ETL_DAG' AIRFLOW_CTX_TASK_ID='copyToSessionTimeStamp' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T19:46:52.815149+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-24T19:46:52.815149+00:00'
[2024-10-24T19:47:27.330+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-24T19:47:27.343+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.4-linuxkit-x86_64-with-glibc2.36
[2024-10-24T19:47:27.347+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-10-24T19:48:06.414+0000] {connectionpool.py:824} WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError("bad handshake: SysCallError(-1, 'Unexpected EOF')"))': /session/v1/login-request?request_id=7a505c20-6b99-409e-89cb-9f874bbe2886&databaseName=dev&schemaName=raw_data&warehouse=stock_warehouse&request_guid=3fde4b0f-8e7f-4da0-86bf-f9a92e1b0b59
[2024-10-24T19:48:08.986+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-10-24T19:48:08.998+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-24T19:48:09.014+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-24T19:48:09.220+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Assignment8_ETL_DAG, task_id=copyToSessionTimeStamp, run_id=manual__2024-10-24T19:46:52.815149+00:00, execution_date=20241024T194652, start_date=20241024T194724, end_date=20241024T194809
[2024-10-24T19:48:09.428+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-24T19:48:09.670+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-24T19:48:09.718+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
